develop-docs/application/config.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 title: Configuration
+sidebar_order: 60
```

Comment:
I spaced most of the pages out by increments of 10 as it was easy to count and left holes for future docs to be added.